go.mod

```diff
@@ -25,6 +23,108 @@ require (
 	k8s.io/utils v0.0.0-20210707171843-4b05e18ac7d9
 )
 
+require (
+	cloud.google.com/go v0.54.0 // indirect
```

Comment:
Why do we need to add so many dependencies? Can you run `go mod tidy` before pushing to remote?

Reply:
Those changes are added automatically after running ```go mod tidy``` @lintongj